learning_wavelets/evaluate.py

```diff
@@ -31,21 +35,18 @@ def keras_ssim(y_true, y_pred):
 
 def psnr_single_image(gt, pred):
     """ Compute Peak Signal to Noise Ratio metric (PSNR) """
+    
     return peak_signal_noise_ratio(gt, pred, data_range=1)
 
 
 def ssim_single_image(gt, pred):
     """ Compute Structural Similarity Index Metric (SSIM).
-
     The images must be in HWC format
     """
-    return structural_similarity(
-        gt, pred, multichannel=True, data_range=1
-    )
+    return structural_similarity(gt[0,...], pred[0,...], multichannel=True, data_range=1)
```

Comment:
this shouldn't be changed, because the images should already be in HWC format.

I think it's because you slice the last dimension in the evaluation script